newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -167,49 +170,57 @@ export const HomePage = React.memo<Props>(
                     onOpenPreferences={onOpenPreferences}
                     onOpenAbout={onOpenAbout}
                   />
-                  {activeTab === 'get-started' && (
-                    <GetStartedSection
-                      onTabChange={setActiveTab}
-                      onCreateProject={() =>
-                        onCreateProject(/*exampleShortHeader=*/ null)
-                      }
-                      onOpenOnboardingDialog={onOpenOnboardingDialog}
-                      showGetStartedSection={showGetStartedSection}
-                      setShowGetStartedSection={setShowGetStartedSection}
-                    />
-                  )}
-                  {activeTab === 'build' && (
-                    <BuildSection
-                      ref={buildSectionRef}
-                      project={project}
-                      canOpen={canOpen}
-                      onChooseProject={onChooseProject}
-                      onOpenProjectPreCreationDialog={
-                        onOpenProjectPreCreationDialog
-                      }
-                      onShowAllExamples={() =>
-                        onCreateProject(/*exampleShortHeader=*/ null)
-                      }
-                      onSelectExample={exampleShortHeader =>
-                        onCreateProject(exampleShortHeader)
-                      }
-                      onOpenRecentFile={onOpenRecentFile}
-                      onChangeSubscription={onChangeSubscription}
-                      storageProviders={storageProviders}
-                    />
-                  )}
-                  {activeTab === 'learn' && (
-                    <LearnSection
-                      onOpenOnboardingDialog={onOpenOnboardingDialog}
-                      onCreateProject={() =>
-                        onCreateProject(/*exampleShortHeader=*/ null)
-                      }
-                      onTabChange={setActiveTab}
-                      onOpenHelpFinder={onOpenHelpFinder}
-                    />
-                  )}
-                  {activeTab === 'play' && <PlaySection />}
-                  {activeTab === 'community' && <CommunitySection />}
+                  <Column
+                    noMargin
+                    expand //TODO: check fullHeight and co on Safari.
```

Comment:
todo?

Reply:
Checked 👍

Reply:
Checked, works well on Safari and Firefox, removing the todo